sdk/src/main/java/io/opentelemetry/sdk/metrics/MeterSdk.java

```diff
@@ -74,4 +75,25 @@
   public BatchRecorder newMeasureBatchRecorder() {
     throw new UnsupportedOperationException("to be implemented");
   }
+
+  @Override
+  public LabelSet createLabelSet(String k1, String v1) {
+    throw new UnsupportedOperationException("to be implemented");
```

Comment:
Consider adding an issue to track this TODO.

Reply:
No implementation is available for the entire metric. I think it is clear that we need to implement that entire package.

https://github.com/open-telemetry/opentelemetry-java/issues/684